src/ImageSharp.Web/Providers/FileProviderImageProvider.cs

```diff
@@ -29,21 +29,26 @@ public class FileProviderImageProvider : IImageProvider
         /// Initializes a new instance of the <see cref="FileProviderImageProvider"/> class.
         /// </summary>
         /// <param name="fileProvider">The file provider.</param>
+        /// <param name="processingBehavior">The processing behavior.</param>
         /// <param name="formatUtilities">Contains various format helper methods based on the current configuration.</param>
-        public FileProviderImageProvider(IFileProvider fileProvider, FormatUtilities formatUtilities)
+        protected FileProviderImageProvider(IFileProvider fileProvider, ProcessingBehavior processingBehavior, FormatUtilities formatUtilities)
```

Comment:
Can the `processingBehavior` parameter move to the end to align with the field/property setting order in the constructor (and also update the XML documentation order):
```suggestion
        protected FileProviderImageProvider(IFileProvider fileProvider, FormatUtilities formatUtilities, ProcessingBehavior processingBehavior)
```

And optionally add a default value?
```suggestion
        protected FileProviderImageProvider(IFileProvider fileProvider, FormatUtilities formatUtilities, ProcessingBehavior processingBehavior = ProcessingBehavior.CommandOnly)
```

Reply:
We don't do default values. They make tracking API changes more difficult.